src/Symfony/Bundle/Resources/config/http_cache_tags.xml

```diff
@@ -5,18 +5,10 @@
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
-        <service id="api_platform.http_cache.purger.varnish" alias="api_platform.http_cache.purger.varnish.ban" public="false" />
```

Comment:
we can't remove existing services it's a backward compatibility break

Reply:
Indeed.